### PR TITLE
#377 stop arcsi

### PR DESCRIPTION
**Pull request by baxgas**
Please check functionality on staging, since local is cors disabled

### CODE REVIEW

**Comment by gammaw** [comment]
> Please check functionality on staging, since local is cors disabled

Can you check it before requesting a review? 

By the way, you can configure to allow CORS locally too. See here: 

https://github.com/mmmnmnm/lahmacun_arcsi/blob/dev/nginx-cors/nginx/https_arcsi.conf.template

**Comment by pvj** [comment]
the code that I can read looks good to me! Would you like us to help w/ testing it out on our test env? What's a good way to test this?

**Comment by gammaw** [comment]
@baxgas I checked out this branch on dev and tried but it didn't seem to work for me. This is what I did: 1) I started an arcsived episode from the player, 2) I then hit play of the "live" radio player. Result: the arcsi player didn't stop and both streams were playing.

**Comment by baxgas** [comment]
@gammaw thank you for checking, there was an error with node targeting type
since I can't make it work locally, I tested it on staging, seems fine (this branch is now on staging)

File: wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue [hunk 1]
```diff
@@ -321,7 +321,8 @@ export default {
             this.audio.play();
             this.is_playing = true;
             document.body.classList.add("Playing");
-            document.querySelectorAll("audio.episodeplay").remove();
+            document.querySelectorAll("audio.episodeplay").forEach(el => el.remove());
+            document.querySelectorAll(".arcsibutton").forEach(el => el.classList.remove('hiddenelement'));
```
Comment (gammaw) [inline wp/wp-content/themes/lahma_maker/assets/playerfiles/radio_player.vue:325]: Looks good!

File: wp/wp-content/themes/lahma_maker/assets/playerfiles/package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "fullcalendar": "^3.10.2",
     "jquery": "^3.5.1",
     "leaflet": "^1.6.0",
-    "lodash": "^4.17.15",
+    "lodash": "^4.17.21",
     "moment": "^2.26.0",
```
Comment (gammaw) [inline wp/wp-content/themes/lahma_maker/assets/playerfiles/package.json:22]: Hopefully, these upgrades won't break anything...

**Review by gammaw — APPROVED** [review]
Nice and clean change! Tested on dev too. 

PS: the upgrades that sneak in with the new vue file builds are a bit scary without a proper regression test/CI system in place but it is what it is..

**Comment by baxgas** [comment]
@gammaw Thank you for the review! you are right the minified radio player.js needs to be regenerated for all vue changes, so the different branches can be a mess
I merge it now

**Pull request merged**